config/packages/prod/deprecations.yaml

```diff
@@ -5,4 +5,4 @@
 #        deprecation:
 #            type: stream
 #            channels: [deprecation]
-#            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
+#            path: php://stderr
```

Comment:
Also - why? Previously we had a kind of a good "source of truth" to watch for the deprecations. How will we do that now?

Reply:
Flex recipe... reverted